src/components/Auth/Registration/ValidationScheme.tsx

```diff
@@ -0,0 +1,30 @@
+import * as Yup from 'yup';
+
+const validationSchema = Yup.object().shape({
+  username: Yup.string()
+    .required('validation.requiredName')
+    .min(3, 'validation.nameMinLength'),
+  age: Yup.number()
+    .required('validation.requiredAge')
+    .min(3, 'validation.ageMinLength')
+    .max(130, 'validation.ageMaxLength'),
+  email: Yup.string()
+    .email('validation.invalidEmail')
```

Comment:
instead of making 1000 new keys, you can make one `invalidField` and pass property like email/age/etc

Reply:
I am sorry, could you explain me better, what should i do and whats wrong with my validation?

Reply:
How many keys/values did u create here? 
Many instead of one generic, am I right ? Invalid email, invalid password, invalid  address, invalid etc.
I guess that in our translation only difference between those new keys/values will be field name.

Reply:
fixed